packages/discord-verify/src/verify.ts

```diff
@@ -1,11 +1,8 @@
-import { webcrypto } from "node:crypto";
-import { TextEncoder } from "node:util";
-
 const encoder = new TextEncoder();
```

Comment:
I'm trusting this works cause I ain't no nothing about this verification process

Reply:
It works, I'm not sure if it works _well_. I figure I can worry about optimizations and potential fixes later